ext/afform/admin/afformEntities/Activity.php

```diff
@@ -1,7 +1,13 @@
 <?php
 return [
   'entity' => 'Activity',
-  'defaults' => "{'url-autofill': '1'}",
+  'defaults' => "{
+    data: {
+      source_contact_id: 'user_contact_id',
```

Comment:
@colemanw does source_contact_id show up in the UI? just wondering if this is changeable and what would user_contact_id reference if its an anon submission?

Reply:
Yes this is the default and can be changed, but I think it's a sensible default. It makes adding a new activity look like this:
![image](https://user-images.githubusercontent.com/2874912/117074051-d4feb680-ad00-11eb-80d4-83530c9b597c.png)

Reply:
(also see **Comments** in PR description about this)